chall05/egarcia-.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+
+#define funtion(x)       printf(x" = %d\n", ft_ie_except_after_c(x));
+
+int     ft_ie_except_after_c(char* str)
+{
+    while (*str)
+    {
+        if (*(str + 2) && *str == 'c' && *(str + 1) == 'i' && *(str + 2) == 'e')
+            return(0);
+        else if (*(str + 1) && *str == 'e' && *(str + 1) == 'i' && *(str - 1) != 'c')
+            return 0;
+        str++;
```

Comment:
You could have used `strstr` to iterate and the different character combination you want to check!  
You would benefit from its optimizations, and would have gained time and readability in your code. I know that in 42's project you are encouraged to recode everything, but in this challenges you are allowed to take full advantage of the language imposed (as long as it remains in standard libraries).